composer.json

```diff
@@ -26,7 +26,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "composer-runtime-api": "^2",
-        "doctrine/dbal": "^3.3",
+        "doctrine/dbal": "^3.4",
```

Comment:
I hope this bump is okay for a bugfix release. If not, I could revert here and skip the three tests I've modified below on the low-deps run.

Reply:
if the supported PHP versions have not changed between DBAL 3.3 and 3.4 (or if the dropped PHP versions were already not supported here), it is fine IMO.

Reply:
And the answer to that is that DBAL 3.4 bumped the min PHP version from 7.3 to 7.4, which is fine as doctrine/migrations already requires PHP 7.4 anyway.